unitTests/ui/Combobox/ComboboxValuesTest.ts

```diff
@@ -119,6 +119,15 @@ export function ComboboxValuesTest() {
         getChildren().forEach(valueElement => expect(valueElement.tabIndex).toEqual(0));
       });
 
+      it('should highlight text that matches the input value', () => {
+        combobox.element.querySelector('input').value = 'e';
+        triggerRenderFromResponse(['hi', 'hello']);
+        expect($$(comboboxValues.element).findAll('.coveo-checkbox-span-label')[0].innerHTML).toBe('hi');
```

Comment:
this expect is not super necessary

Reply:
Right. I wanted to test that if the regex does not match, it would not be highlighted. but, when thinking about it, if it doesn't match the regex, it wouldn't even be displayed as a value.

Removed it. I'll delete it from the argument list from  `triggerRenderFromResponse ` as well